### ISSUE TITLE
No output when build fails

### ISSUE THREAD

**Issue by Graham42** [MoOx/eslint-loader#152]
With `failOnError` set to `true` the build properly fails, but doesn't report the eslint report that caused it to fail. Just outputs this stacktrace.
```
$ webpack --progress --profile --colors --bail

 62% 942/1071 build modulesError
    at NormalModule.onModuleBuildFailed (/<project_dir>/node_modules/webpack-core/lib/NormalModuleMixin.js:315:19)
    at nextLoader (/<project_dir>/node_modules/webpack-core/lib/NormalModuleMixin.js:270:31)
    at /<project_dir>/node_modules/webpack-core/lib/NormalModuleMixin.js:292:15
    at runSyncOrAsync (/<project_dir>/node_modules/webpack-core/lib/NormalModuleMixin.js:173:4)
    at nextLoader (/<project_dir>/node_modules/webpack-core/lib/NormalModuleMixin.js:290:3)
    at /<project_dir>/node_modules/webpack-core/lib/NormalModuleMixin.js:259:5
    at Storage.finished (/<project_dir>/node_modules/enhanced-resolve/lib/CachedInputFileSystem.js:38:16)
    at /<project_dir>/node_modules/graceful-fs/graceful-fs.js:78:16
    at FSReqWrap.readFileAfterClose [as oncomplete] (fs.js:415:3)
error Command failed with exit code 1.

```

**Comment by rjbultitude** [comment]
Hi @Graham42 
I had a similar issue, which was fixed by explicitly setting `emitError` to `true`. 
I'm using version 1.3.0 and WebPack 2.2.0

**Comment by MoOx** [comment]
By default it should emit eslint errors as webpack errors and warnings as warnings. Build will fails depending on webpack config... Not sure what to do here.

**Comment by Graham42** [comment]
Updated to webpack 2, things are working now.
